### PR TITLE
17 02 2021

### PR DESCRIPTION
**Pull request by BlakeTurner**
Reduced readme to what we actually have, added "how to quit" guide.

### CODE REVIEW

File: how-to-quit.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+1. Don't keep it a secret! 
+
+Have a real conversation with your manager about where you're at. Maybe there's a conversation we can have that changes your your mind. If not, maybe we can support you in seeking your next role. Regardless, it's hugely beneficial for us to see Bench through your eyes so we can continue to improve.
+
+2. Respect your teammates and their commitments. 
+
+If you're working towards a commitment that your departure would impact, take that into account before setting any timelines. If you're a manager or more senior in general, consider a longer notice period (1 month+) so that we can facilitate a smooth transition.
+
+_That's it._ After that, it's about getting the most out of your remaining time, and thanking you for what you've contributed thus far. We could not have gotten to where we are without your help.
```
Comment (vweller) [inline how-to-quit.md:13]: I love the tone and the message in this new section! Thank you.
Should we include the ticket that needs to be created in People Team space? https://benchco.atlassian.net/servicedesk/customer/portal/31/group/150/create/631

Comment (BlakeTurner) [reply how-to-quit.md:13]: I don't want to be tied to URLs that might change, but I'll put a note in about what to do here, thanks for the callout

**Review by vweller — APPROVED** [review]

File: how-were-organized.md [hunk 2]
```diff
@@ -32,7 +33,15 @@ The Platform team owns our cloud infrastructure, our build pipeline, and our dev
 
 #### Core
 
-The Core team maintains and improves our in-house software and its infrastructure. The team's primary concern is Keeping the Lights On (KTLO) at the application level, their secondary concern is supporting stakeholder requests that are "too small for a pitch", and their tertiary concern is innovating our technology and processes to enhance the speed and quality of our engineering, with a strong focus on the reduction of technical debt. The Core team works closely with BAS (Bench App Support) and the rest of BizTech to handle the issues, outages, and stakeholder requests that come our way during a cycle and its cooldown. In effect, the Core team makes the Product Engineering cycles _possible_ by absorbing the myriad distractions the team needs to deal with during a cycle. The number of engineers on the Core team scales based on total number of clients, a proxy for business complexity.
+The Core group maintains and improves our in-house software and its infrastructure. They are in two teams: Core Support, and Core Systems.
```
Comment (vweller) [inline how-were-organized.md:36]: "They are in two teams: Core Support, and Core Systems."
This reads a bit choppy.
Suggest: They are organized in two teams: Core Support, and Core Systems.

File: how-were-organized.md [hunk 3]
```diff
@@ -32,7 +33,15 @@ The Platform team owns our cloud infrastructure, our build pipeline, and our dev
...
+
+##### Core Support
+
+The Core Support team's primary concern is Keeping the Lights On (KTLO) at the application level, their secondary concern is supporting stakeholder requests that are "too small for a pitch", and their tertiary concern is innovating to make both KTLO and supporting stakeholders more efficient. Core Support works closely the rest of BizTech to handle the issues, outages, and stakeholder requests that arise during our build cycles. In effect, the Core Support team makes the Product Engineering cycles _possible_ by absorbing the myriad distractions that would otherwise derail their focus. The number of engineers on the Core Support team scales based on total number of clients, a proxy for business complexity.
```
Comment (vweller) [inline how-were-organized.md:40]: "Core Support team scales based on total number of _clients_"
shouldn't this be **clients & Benchmates?**

File: how-were-organized.md [hunk 4]
```diff
@@ -32,7 +33,15 @@ The Platform team owns our cloud infrastructure, our build pipeline, and our dev
...
+
+##### Core Systems
+
+The Core System team is focused on improving our technology and processes to enhance the speed and quality of our engineering, with a strong focus on the reduction of technical debt. Unlike Core Support, they work with few distractions in order to increase the speed with which they can make improvements.
```
Comment (vweller) [inline how-were-organized.md:44]: Here we are missing how we scale this team. 
My thought is that is based on complexity & amount of tech debt... but how do we quantify that?

**Review by vweller — CHANGES_REQUESTED** [review]

File: how-were-organized.md (hunk 3 above)
Comment (BlakeTurner) [reply how-were-organized.md:40]: Benchmates tend to scale w/ Clients. This is the simplest proxy we could find.

File: how-were-organized.md (hunk 4 above)
Comment (BlakeTurner) [reply how-were-organized.md:44]: I think we just leave it blank for now 😉

**Review by vweller — APPROVED** [review]
Thanks! 👍

**Pull request merged**